controllers/resource/resource_generator.go

```diff
@@ -164,6 +179,59 @@ func (r *resourceGenerator) GetPodTemplate(doguResource *k8sv1.Dogu, dogu *core.
 	return podTemplate, nil
 }
 
+func getChownInitContainer(dogu *core.Dogu, doguResource *k8sv1.Dogu) (*corev1.Container, error) {
```

Comment:
Maybe `get` implies the wrong things here, because far more happens inside this function, namely building the init command chain for creating and re-owning directories in a dogu's volume. 

I know naming is hard, and yet I feel we should ponder a bit more over this function's name.
Maybe it is also a hint that here happens to much in a single place, but I feel the granularity is fine.